pkg/k8sutil/portforward.go

```diff
@@ -102,5 +109,181 @@ func PortForward(kubeContext string, localPort int, remotePort int, namespace st
 		time.Sleep(time.Millisecond * 100)
 	}
 
+	if pollForAdditionalPorts {
+		type AdditionalPort struct {
+			ServiceName string `json:"serviceName"`
+			ServicePort int    `json:"servicePort"`
+			LocalPort   int    `json:"localPort"`
+		}
+
+		forwardedAdditionalPorts := map[AdditionalPort]chan struct{}{}
+
+		uri := fmt.Sprintf("http://localhost:%d/api/v1/kots/ports", localPort)
+		go func() error {
+			for {
```

Comment:
This go func is weird.  It can terminate with an error, but it doesn't inform anything that it has terminated. Also errors are thrown away